app/app.go

```
@@ -993,11 +994,15 @@ func (app *App) preBlocker(ph *ProposalHandler) func(sdk.Context, *abci.RequestF
 }
 
 func (app *App) RegisterUpgradeHandlers() {
-	const UpgradeName = "v0.3.0"
+	const UpgradeName = "v0.4.1"
 
 	app.UpgradeKeeper.SetUpgradeHandler(
 		UpgradeName,
 		func(ctx context.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+			sdkctx := sdk.UnwrapSDKContext(ctx)
+			params := app.GlobalFeeKeeper.GetParams(sdkctx)
+			params.MinimumGasPrices = sdk.NewDecCoins(sdk.NewDecCoinFromDec(BondDenom, math.LegacyNewDecWithPrec(25, 4)))
+			_ = app.GlobalFeeKeeper.SetParams(sdkctx, params)
```

maybe don't ignore the error